TFlippy_TerritoryControl_Items_Dev/Entities/Items/Drugs/Boof/Boofed.as

```diff
@@ -48,8 +48,17 @@ void onTick(CBlob@ this)
 			
 				ShakeScreen2(5.0f, 5.0f, this.getPosition());
 				if (getGameTime() % 5 == 0) SetScreenFlash(50, 10, 60, 50);
-			
-				if (XORRandom(500) == 0)
+				
+				if (XORRandom(300) == 0)
+				{
+					if (isClient()) 
```

Comment:
You should do isClient() && XORRandom(300) on the same if statement

Reply:
@Vam-Jam fixed. There was a duplicate of isClient which didn't do anything.